src/lib/components/header.svelte

```diff
@@ -2,8 +2,8 @@
   <div class="header">
     <nav>
       <a href="/"> Hjem </a>
-      <a href="hola"> Bodegaer </a>
-      <a href="hola"> Producenter </a>
+      <a href="/"> Bodegaer </a>
```

Comment:
why the space before the text ? this is something you do a lot, but its not consistent so should never be done